From 47b6d6249df2bd226b857643377332fcd62c1989 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Thu, 2 Sep 2010 18:11:31 +0000
Subject: [PATCH 13/25] gettextize: git-merge remaining cmd_merge messages
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Translate "Already up-to-date.", "Fast-forward", "no commit created"
and other cmd_merge() messages.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/merge.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/builtin/merge.c b/builtin/merge.c
index 9121c7b..ce831bf 100644
--- a/builtin/merge.c
+++ b/builtin/merge.c
@@ -1078,7 +1078,7 @@ int cmd_merge(int argc, const char **argv, const char *prefix)
 		 * If head can reach all the merge then we are up to date.
 		 * but first the most common case of merging one remote.
 		 */
-		finish_up_to_date("Already up-to-date.");
+		finish_up_to_date(_("Already up-to-date."));
 		return 0;
 	} else if (allow_fast_forward && !remoteheads->next &&
 			!common->next &&
@@ -1095,10 +1095,10 @@ int cmd_merge(int argc, const char **argv, const char *prefix)
 				hex,
 				find_unique_abbrev(remoteheads->item->object.sha1,
 				DEFAULT_ABBREV));
-		strbuf_addstr(&msg, "Fast-forward");
+		strbuf_addstr(&msg, _("Fast-forward"));
 		if (have_message)
 			strbuf_addstr(&msg,
-				" (no commit created; -m option ignored)");
+				_(" (no commit created; -m option ignored)"));
 		o = peel_to_type(sha1_to_hex(remoteheads->item->object.sha1),
 			0, NULL, OBJ_COMMIT);
 		if (!o)
@@ -1156,7 +1156,7 @@ int cmd_merge(int argc, const char **argv, const char *prefix)
 			}
 		}
 		if (up_to_date) {
-			finish_up_to_date("Already up-to-date. Yeeah!");
+			finish_up_to_date(_("Already up-to-date. Yeeah!"));
 			return 0;
 		}
 	}
-- 
1.7.2.2.614.g1dc9

